src/libs/Visibility/index.desktop.ts

```diff
@@ -1,31 +1,19 @@
 import ELECTRON_EVENTS from '../../../desktop/ELECTRON_EVENTS';
+import {HasFocus, IsVisible, OnVisibilityChange} from './types';
 
 /**
  * Detects whether the app is visible or not. Electron supports document.visibilityState,
  * but switching to another app while Electron is partially occluded will not trigger a state of hidden
  * so we ask the main process synchronously whether the BrowserWindow.isFocused()
- *
- * @returns {Boolean}
  */
-function isVisible() {
-    return window.electron.sendSync(ELECTRON_EVENTS.REQUEST_VISIBILITY);
-}
+const isVisible: IsVisible = () => !!window.electron.sendSync(ELECTRON_EVENTS.REQUEST_VISIBILITY);
```

Comment:
I don't think we had to remove the named function here.

`function isVisible(): boolean` would be fine I guess.

Reply:
From TS guidelines:

> In modules with platform-specific implementations, create `types.ts` to define shared types. For platform-specific implementations, always define shared types that complies with all variants.

That's why `IsVisible` type was created in the first place and used here. I think the change was necessary.